### ISSUE TITLE
Animation Track Duplication

### ISSUE THREAD

**Issue by ExquisiterEmil** [godotengine/godot-proposals#6205]
### Describe the project you are working on

A 2D rythm-platformer (it's complicated)

### Describe the problem or limitation you are having in your project

I have an Animation Player that edits property X of Node A. Now I have a Node B of the same type, which I would also want to have that animation. The current way to do this, is to make a new AnimationTrack for Node B and manually set the same keyframes as I have in Node A.
![image](https://user-images.githubusercontent.com/53947784/215363797-fe3efb3d-81df-412b-bf7c-dd7452839aec.png)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

It would be nice to have the option to duplicate an entire animation track, just with a different path that it modifies. Same keyframes at the same time.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

When right clicking on the header or path of a Track, a context menu should show, with a duplicate action:
![image](https://user-images.githubusercontent.com/53947784/215363395-e8c4e772-b08b-4eb7-848d-1b2b2ebf9194.png)
Upon clicking this action, or alternatively/additionally, when selecting a Track and Pressing CTRL+D, a "Select Node" menu should show, where the user can select a different Node.
![image](https://user-images.githubusercontent.com/53947784/215363540-0f29e71e-a75f-4777-bbe6-ae8b3e4fc4e5.png)
Then, a new Animation Track gets added with the path to NodeA being replaced with the path to NodeB.
![image](https://user-images.githubusercontent.com/53947784/215363839-faa41dbc-bd9d-4d74-9bfd-02fc80319a16.png)

### If this enhancement will not be used often, can it be worked around with a few lines of script?

I think, this would benefit the AnimationPlayer usability a lot. It could maybe be achieved with a plugin, but I think it would be very clunky.

### Is there a reason why this should be core and not an add-on in the asset library?

AnimationPlayer is core and its usability is important :)

**Comment by KoBeWi** [comment]
You can copy the tracks using the Edit menu. However there is no easy way to change the track path (#4847 covers that)

**Comment by ExquisiterEmil** [comment]
@KoBeWi Indeed, I forgot you can do that. That already makes it a little easier, although I never found this Edit menu particularly intuitive. Your proposal would be very useful as well!

**Comment by digilena** [comment]
I went looking to see if this was already noted as a bug. I am also facing the same issue. I want to implement the same animation track to two different nodes and change the X coordinate for the second one. Every time I modify one node, it modifies the other due to the path (I even change which Root Node it points to, but this doesn't seem to matter).
